onnxruntime/core/providers/webnn/builders/impl/gru_op_builder.cc

```diff
@@ -135,12 +135,13 @@ bool GruOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializers, c
   int32_t steps = static_cast<int32_t>(input_shape[0]);
 
   if (TensorExists(input_defs, 4)) {
-    if (!Contains(initializers, input_defs[4]->Name())) {
+    const auto* seq_initializer = graph_viewer.GetConstantInitializer(input_defs[4]->Name());
+    if (!seq_initializer) {
```

Comment:
Nit: to keep consistent with `lstm_op_builder.cc`
```suggestion
    const auto* sequence_lens_init = graph_viewer.GetConstantInitializer(input_defs[4]->Name());
    if (!sequence_lens_init ) {
```